app/src/main/java/org/simple/clinic/patient/sync/PatientPayload.kt

```diff
@@ -117,7 +117,9 @@ data class PatientAddressPayload(
   fun toDatabaseModel(): PatientAddress {
     return PatientAddress(
         uuid = uuid,
+        houseAddress = null,
```

Comment:
Should we add a `TODO` here to read this from the api?

Reply:
Nice catch. Added.